cg/apps/scout/scout_export.py

```diff
@@ -47,20 +60,22 @@ class Gene(BaseModel):
 
 
 class DiagnosisPhenotypes(BaseModel):
+    model_config = ConfigDict(coerce_numbers_to_str=True)
     disease_nr: int
     disease_id: str
     description: str
     individuals: Optional[List[Dict[str, str]]] = None
 
 
 class ScoutExportCase(BaseModel):
+    model_config = ConfigDict(coerce_numbers_to_str=True)
     id: str = Field(str, alias="_id")
     analysis_date: datetime
     owner: str
     causatives: Optional[List[str]] = None
     collaborators: List[str] = []
     individuals: List[Individual]
-    genome_build: Annotated[str, BeforeValidator(lambda x: str(x))] = GENOME_BUILD_37
+    genome_build: Annotated[str, BeforeValidator(convert_genome_build)] = GENOME_BUILD_37
```

Comment:
Is the before validator necessary here?

Reply:
Yes - if we want to consider cases where a dict is provided where genome_build is set to None.